.github/workflows/ci.yml

```diff
@@ -24,7 +24,7 @@ env:
   BYOND_MINOR: 1566
 
   #This can be declared here because build.py will ignore it if DM_UNIT_TESTS is true.
-  ALL_MAPS: tgstation metaclub defficiency packedstation roidstation nrvhorizon test_box test_tiny snaxi tgstation-sec tgstation-snow LampreyStation xoq boxesstation synergy bagelstation lowfatbagel Dorfstation waystation island
```

Comment:
This apparently removed Island from CI, it didn't add linestation.

Reply:
A mix up at the cookie factory, it should be resolved now

Reply:
> This apparently removed Island from CI

The hero we didn't deserve but the one we needed.